tests/foreman/cli/test_computeresource_libvirt.py

```diff
@@ -49,6 +49,8 @@
 from robottelo.datafactory import parametrized
 from robottelo.decorators import skip_if_not_set
 
+pytestmark = pytest.mark.libvirt_content_host
```

Comment:
Bad, not whole module requires provisioning set up, there are tier1,2 tests trying to insert 256 chars long strings - no vlans invloved

Reply:
No, wrong, even though they are Tier1/2 , they are trying to connect to the libivrt CR from our old infra. 

I we run those test cases in new infra, they will fail to make contact and hence they have to be excluded with that marker.